content/en/docs/armory-admin/terraform-enable-integration.md

```diff
@@ -37,16 +37,16 @@ Note that all Terraform stages within a Pipeline that affect state must use the
   * GitHub
   * BitBucket
   * HTTP artifact
-
-Although not required, Armory recommends an external Redis instance for the Terraform Integration. For more information, see [Redis](#redis).
+* Dedicated external Redis instance
+  * Armory requires configuring a dedicated external Redis instance for production usage of the Terraform Integration. This is to ensure that you do not encounter scaling or stability issues in production. For more information [Redis](#redis).
```

Comment:
Armory the company or Armory Enterprise the product??

Reply:
The company. Technically, you can use the integration without an external redis, but we won't support it.